src/usr/local/www/vpn_ipsec_settings.php

```diff
@@ -68,6 +70,33 @@
 		}
 	}
 
+	$bypassrules = array();
+	for ($x = 0; $x < 99; $x++) {
+		if (isset($_POST["source{$x}"]) && isset($_POST["destination{$x}"]) &&
+		    isset($_POST["srcmask{$x}"]) && isset($_POST["dstmask{$x}"])) {
+			$source = htmlspecialchars($_POST["source{$x}"]) . '/' . htmlspecialchars($_POST["srcmask{$x}"]);
+			$destination = htmlspecialchars($_POST["destination{$x}"]) . '/' . htmlspecialchars($_POST["dstmask{$x}"]);
+			$rule = array();
+			$rule['source'] = htmlspecialchars($_POST["source{$x}"]);
+			$rule['srcmask'] = htmlspecialchars($_POST["srcmask{$x}"]);
+			$rule['destination'] = htmlspecialchars($_POST["destination{$x}"]);
+			$rule['dstmask'] = htmlspecialchars($_POST["dstmask{$x}"]);
+			$bypassrules['rule'][] = $rule;
```

Comment:
This could all be done in one assignment

    $bypassrules['rule'][] = array(
        ...
    );